open-company/open-company-web
# Cancel subscription

#### Review after #1054 

---

Card: https://trello.com/c/ZFQWmeka

To test:
- signup for a new org
- go to billing panel
- add a payment method
- [x] check on stripe that you have an invoice for the initial plan
- now go to billing panel again
- click cancel subscription
- [x] do you get a modal to confirm the subscription cancel? (check the copy)
- click "No, keep it"
- [x] nothing happened?
- click cancel subscription again
- click "Yes, cancel it" in the modal
- [x] do you see a spinner besides "Cancel subscription"?
- wait for the spinner to end
- [x] do you see the right copy now (saying when the cancellation will take place)?
- refresh
- open billing panel
- [x] do you see the right copy?
- click "Subscribe to Carrot"
- [x] switch the plan dropdown and make sure the button is always enabled (you can restart on both plans, no matter which one your are on)
- pick a plan
- click "Subscribe to Carrot"
- wait for the request to complete
- [x] check the copy is now ok
